templates/main_screen.mustache

```diff
@@ -1,38 +1,30 @@
 <div class="container-fluid mootimetercontainer {{{containerclasses}}}">
     <div class="row h-100">
         {{#showpagescol}}
-        <div class="mootimetercolpages col-md-2 col-sm-3 col-xs-12 ">
-
-            <div>
-                <ul style="padding-left: 0; margin-top:10px;">
-                    {{#pages}}
-                    <li class="list-group-item mootimeter_pages_li {{active}}" data-pageid="{{pageid}}" data-cmid="{{cmid}}">
-                        <img src="{{{pix}}}" width="20px">
-                        {{title}}
-                    </li>
-                    {{/pages}}
-                    {{#isediting}}
-                    <li id="mootimeter_addpage" class="list-group-item mootimeter_addpage" data-action="addpage" data-cmid="{{cmid}}">
-                        <i class="fa fa-plus"></i> {{#str}} addpage, mod_mootimeter {{/str}}
-                    </li>
-                    {{/isediting}}
-                </ul>
-            </div>
+        <div class="mootimetercol mootimetercolpages">
+                {{#pages}}
+                <a href="?id={{cmid}}&pageid={{pageid}}" class="list-group-item mootimeter_pages_li py-0 px-2 d-flex {{active}}" data-pageid="{{pageid}}" data-cmid="{{cmid}}">
```

Comment:
I still prefer to use <ul><li> here, then perspectively the elements should be able to be sorted by drag. A.f.a.i.k. this would not be possible with a-tags or divs.

Reply:
Regardless, I like the look of what you've done with the site map. I would very much like to incorporate that into the `<ul><li>`.